Uplift/Controllers/GymDetailViewController.swift

```diff
@@ -51,7 +51,7 @@ class GymDetailViewController: UIViewController {
         self.gymDetail = GymDetail(gym: gym)
 
         for facility in gymDetail.facilities where facility.name == "Fitness Center" {
-            self.equipment = categorizeEquipment(equipmentList: facility.equipment)
+            self.equipment = categorizeEquipment(equipmentList: facility.details[0].equipment)
```

Comment:
Is there guaranteed to be only 1 item in details? what even is this lmao

Reply:
if the facility is Fitness Center then details[0] is always gonna be the facilitydetail that corresponds to equipment

the structure has to do with the way the models are done

Reply:
To be more typesafe, `facility.details.first?.equipment ?? < Default Value >`?